text/0000-data-formats.md

```diff
@@ -0,0 +1,151 @@
+
+
+### Easily Extensible
+
+Not all semantics conventions and common operations are known at this time. It must be possible to add more semantic definitions without creating to versioning or compatibility issues with OpenTelemetry implementations.
+
+
+## Examples of Common Operations
+
+#### RPC calls
+
+Modeling distributed transactions is a core usecase, it is common to analyze  errors and latency for RPC calls. Because multiple individual network requests may be involved in a single logical RPC operation, it must be possible to describe RPC calls as logical operations, independent from the descriptions of transport and application protocols which they contain.
+
+Examples:
+*   A client sending a request to a remote server.
+*   A server receiving a request from a remote client.
+*   Logical vs physical networking: a request for “[https://www.wikipedia.org/wiki/Rabbit](https://www.wikipedia.org/wiki/Rabbit)” results in two physical HTTP requests - one 301 redirection, and another 200 okay. The  logical RPC operation represents both requests together.
+
+#### Network protocols
+
+The internet suite of network protocols is thoroughly standardized, and the details of network interactions calls are critical to analysing the behavior of distributed systems.
+
+Examples:
+*   Application Protocols: HTTP/1.1, HTTP/2, gRPC
+*   Transport Protocols: TCP, UDP
+*   Internet Protocols: IPv4, IPv6
+
+#### Database protocols
+
+Database requests often contain information which is key to root causing errors and latency.
+
+Examples:
+*   SQL: MySQL, Postrgres
+*   NoSQL: MongoDB
+
+
+# Logical Structure
+
+OpenTelemetry observability model, defined as data structures instead of API calls. A logical layer describes the default data structures expected to be produced by OpenTelemetry APIs for transport.
+
+## Requirements
+
+### Support Metadata
+
+The logical structure must be able to support all of the requirements for the metadata layer.
+
+
+### Support both an API and a Streaming Representation
+
+OpenTelemetry data structures must work well with the APIs and SDKs which produce them. It must be possible to stream data from services without resorting to an encoding or exchange protocol which bears little resemblance to the API. 
+
+
+# Encoding Formats
+
+While there is a single set of OpenTelemetry data structures, they can be encoded in a variety of serialization formats.
+
+## Requirements
+
+### Prefer pre-existing formats
+
+Codecs must be commonly available in all major languages. It is outside the bounds of the OpenTelemetry project to define new serialization formats.
+
+**Examples:**
+*   Protobuf
+*   JSON
+
+
+# Exchange Protocols
+
+Exchange protocols describe how to transmit serialized OpenTelemetry data. Beyond just defining which application and transport protocol should be used, the exchange protocol defines transmission qualities such as streaming, retry, acks, backpressure, and throttling.
```

Comment:
@tedsuo now that you've mentioned `retry`, I wonder if it'll be interesting to put `de-dupe` - they are like twins.